powerauth-webflow-i18n/src/main/java/io/getlime/security/powerauth/app/webflow/i18n/I18NService.java

```diff
@@ -38,6 +41,9 @@ public class I18NService {
     @Resource
     private ReloadableResourceBundleMessageSourceWithListing messageSource;
 
+    @Resource
```

Comment:
🤔 I am wondering, why `jakarta.annotation.Resource` instead of `@Autowired` is used, but we may fix it in another PR.